src/ducks/recurrence/service.js

```diff
@@ -0,0 +1,100 @@
+import omit from 'lodash/omit'
+import groupBy from 'lodash/groupBy'
+import keyBy from 'lodash/keyBy'
+import { log } from './logger'
+
+import { Q } from 'cozy-client'
+
+import { TRANSACTION_DOCTYPE } from 'doctypes'
+import { findAndUpdateRecurrences } from './search'
+import { fetchHydratedBundles, saveHydratedBundles } from './api'
+import { getLabel } from './utils'
+
+import addDays from 'date-fns/add_days'
+
+const NB_MONTH_LOOKBACK = 3
+
+const makeQueryForTransactions = recurrences => {
+  if (recurrences.length === 0) {
+    return Q(TRANSACTION_DOCTYPE)
+  } else {
+    const lookbackDateLimit = addDays(new Date(), -NB_MONTH_LOOKBACK * 30.5)
```

Comment:
What is this 30.5 const?

Reply:
I named the constant `DAYS_IN_MONTH`. Thanks.